apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -306,7 +306,7 @@ const TutorialExplorer = React.createClass({
 });
 
 
-function getFilters(robotics) {
+function getFilters(options) {
```

Comment:
Worth documenting what fields we expect on options in a comment? alternatively, you can use ES6 to sort of self-document this for you, i.e.
```js
function getFilters({robotics, mobile}) {
  // elsewhere
  if (mobile) {
     ...
 }
}